core/src/main/java/org/infinispan/persistence/manager/PersistenceManagerStub.java

```diff
@@ -34,6 +34,8 @@
 @Scope(Scopes.NAMED_CACHE)
 @SurvivesRestarts
 public class PersistenceManagerStub implements PersistenceManager {
+   private static final CompletableFuture<Long> SIZE_ZERO_FUTURE = CompletableFuture.completedFuture(0L);
```

Comment:
Seems probably useful to promote this to CompletableFutures, wdyt?

Reply:
This is the only user so far: for some old-forgotten reason, `PersistenceManagerImpl` returns -1L when there's no store, and `PersistenceManagerStub` returns 0L all the time.

Actually Gustavo removed all usages of `PersistenceManagerStub` from production code, so I think I'll just delete the class.